core/testcontainers/core/generic.py

```diff
@@ -69,7 +70,20 @@ def start(self) -> "DbContainer":
         self._configure()
         super().start()
         self._connect()
+        self._seed()
         return self
 
     def _configure(self) -> None:
         raise NotImplementedError
+
+    def _seed(self) -> None:
+        raise NotImplementedError
```

Comment:
Just realized we of course shouldn't penalize DbContainer variants for not defining this method, should do something like `if not self.seeds: return` and only otherwise raise this error